go/git/put.go

```diff
@@ -0,0 +1,53 @@
+package git
+
+import (
+	"context"
+	"encoding/base64"
+
+	"github.com/keybase/client/go/chat/globals"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-codec/codec"
+)
+
+func PutMetadata(ctx context.Context, g *globals.Context, arg keybase1.PutGitMetadataArg) error {
+	// TODO: replace these with real implementations
+	var teamer Teamer
+	var cryptoer Cryptoer
+
+	teamIDVis, err := teamer.LookupOrCreate(ctx, arg.Folder)
+	if err != nil {
+		return err
+	}
+
+	// TODO: version this struct
+	mh := codec.MsgpackHandle{WriteExt: true}
+	var msgpackLocalMetadata []byte
+	enc := codec.NewEncoderBytes(&msgpackLocalMetadata, &mh)
```

Comment:
we have to version anything we encrypt. This data needs to be put into a versioned wrapper.